examples/cluster-configs/trn1-16-nodes-pcluster.md

```diff
@@ -95,7 +101,7 @@ Where
 This will create a ParallelCluster in your AWS account, and you may inspect the progress in AWS CloudFormation console.
 
 
-3. After the cluster is created successfully, perform the [post-install actions](../general/ami/post_installation.md) to configure the head node and compute nodes. 
+3. After the cluster is created successfully, [post-install actions](../general/ami/post_installation.md) now takes place automatically via `CustomActions` indicated in `launch.yaml` to configure the head node and compute nodes. 
```

Comment:
This happens as part of the creation. "While the cluster is being setup, [post-install actions](../general/ami/post_installation.md) now takes place automatically via `CustomActions` indicated in `launch.yaml` to configure the head node and any static compute nodes (`MinCount` > 0).  "

Reply:
added in commit https://github.com/aws-neuron/aws-neuron-parallelcluster-samples/pull/14/commits/58a0bb9c5eb24dbfa81b4b6ad88b8ac125baa3cd